<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppendDeviceWindow</class>
 <widget class="QWidget" name="AppendDeviceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1196</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="MainWidget" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>861</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color:#00000000;
}</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>861</width>
      <height>501</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color:#ffffff;
	border:0px outset #cccccc;
	border-radius:00px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>861</width>
      <height>101</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color:#ffffff;
	border:0px outset #cccccc;
	border-radius:50px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="titleArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>861</width>
      <height>611</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color:#990000ff;
	color:#ffffff;
	border:10px outset #ff0000ff;
	border-radius:50px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="titleTextarea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>861</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777214</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	color:#ffffff;
}</string>
    </property>
    <property name="text">
     <string>添加虚拟设备</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>选择使用的MOD包:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectFileButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>150</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#990000ff;
	color:#ffffff;
	border:1px outset #ff0000ff;
	border-radius:50px;
}
QPushButton::hover{
	background-color:#99000099;
	
}
QPushButton::pressed{
	color:#999999;
}</string>
    </property>
    <property name="text">
     <string>选择</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="selectEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>721</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color:#ffffff;
	color:#000000;
	border:1px outset #ff999999;
	border-radius:50px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>MOD基本信息</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>20</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color:#ffffff;
}</string>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>530</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#990000ff;
	color:#ffffff;
	border:1px outset #ff0000ff;
	border-radius:50px;
}
QPushButton::hover{
	background-color:#99000099;
	
}
QPushButton::pressed{
	color:#999999;
}</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>530</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#990000ff;
	color:#ffffff;
	border:1px outset #ff0000ff;
	border-radius:50px;
}
QPushButton::hover{
	background-color:#99000099;
	
}
QPushButton::pressed{
	color:#999999;
}</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="stateInfo">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	color:#ff0000;
}</string>
    </property>
    <property name="text">
     <string>&lt;未选择包&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>220</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>设备配置信息</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>类型:</string>
    </property>
   </widget>
   <widget class="QLabel" name="DeviceTypeText">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>无效MOD</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="deviceFunctionInfoText">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>291</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color:#ffffff;
	color:#000000;
	border:0px outset #ff0000ff;
	border-radius:50px;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:15pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;无&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>功能:</string>
    </property>
   </widget>
   <widget class="QWidget" name="DevConfigPage" native="true">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>431</width>
      <height>271</height>
     </rect>
    </property>
    <widget class="QLabel" name="deviceIcon">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>231</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="DeviceNameEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color:#ffffff;
	color:#000000;
	border:1px outset #ff999999;
	border-radius:50px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>设备名:</string>
     </property>
    </widget>
    <widget class="QWidget" name="deviceAddress" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>0</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>占用地址:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="useMemBegin">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color:#ffffff;
	color:#000000;
	border:1px outset #ff999999;
	border-radius:50px;
}</string>
      </property>
      <property name="text">
       <string>0x00000000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="useMemEnd">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0x00000000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>243</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="MemAllocFailProm">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	color:#ff0000;
}</string>
     </property>
     <property name="text">
      <string>找不到可分配的内存空间</string>
     </property>
    </widget>
    <widget class="QWidget" name="InterNum" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>中断号:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="interNumTextArea">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color:#ffffff;
	color:#000000;
	border:1px outset #ff999999;
	border-radius:50px;
}</string>
      </property>
      <property name="text">
       <string>000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>titleArea</zorder>
   <zorder>label_3</zorder>
   <zorder>titleTextarea</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>selectFileButton</zorder>
   <zorder>selectEdit</zorder>
   <zorder>label_6</zorder>
   <zorder>line</zorder>
   <zorder>okButton</zorder>
   <zorder>cancelButton</zorder>
   <zorder>stateInfo</zorder>
   <zorder>label_7</zorder>
   <zorder>label_11</zorder>
   <zorder>DeviceTypeText</zorder>
   <zorder>deviceFunctionInfoText</zorder>
   <zorder>label_12</zorder>
   <zorder>DevConfigPage</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
